packages/ui/components/core/Footer.tsx

```diff
@@ -0,0 +1,91 @@
+import { Container, Title, Grid, Text, Stack, Group, createStyles } from '@mantine/core'
+
+import { Button } from './Button'
+import { SocialMediaIconButton } from './SocialMediaIconButton'
+import { BodyGrid } from '../layout'
+
+const useStyles = createStyles((theme) => ({
+	footer: {
+		backgroundColor: theme.other.colors.primary.footer,
+		['& a']: {
+			color: theme.colors.dark,
```

Comment:
we are using the color definitions in the `other` object of the theme

```suggestion
			color: theme.other.colors.secondary.black,
```